acct-group/tox/Manifest

```diff
@@ -0,0 +1,2 @@
+EBUILD tox-0.ebuild 250 BLAKE2B 5842cc4a96b1d430078c2d38c25f0ee62fb62436fed2c886d8c412aaa0e9bb044f898b269cb6a299aa17c4004f59f15d405f1879c6b41eab449ae0d9c8d5f03e SHA512 9fc96402e37aa6e9dfa9ed98469300c3683b70b1b8ac879c29aeb3f18ef485972880e5bbe2a35933e33546d6f4258d42d411e9ba98944f3ce777ced90356c801
```

Comment:
Git wont allow using thin-manifests, please do the work in your git repository. Run repoman.